Source/src/ui/ImGuiUtils.h

```diff
@@ -19,6 +19,11 @@ namespace Hachiko
 
         bool CollapsingHeader(GameObject* game_object, Component* component, const char* header_name);
 
+        bool ToolbarButton(ImFont* font, const char* font_icon, bool active, const char* tooltip_desc);
```

Comment:
As I tested, `ImGui::PushFont` does not accept `const ImFont*` but to keep the code more const-correct, we can make it `ImFont* const font` (which makes the address const) instead so that we permit things such as doing `font = new ImFont();` inside ToolbarButton

Reply:
I was a bit hesitant at first since I try to stick to the demo when doing anything ImGui related as much as possible but I tried it out and it seems to not give any problems. Since we are not changing the pointer in this function I think it's a good addition, thanks!

Reply:
I agree, thanks!